webauthn-server-attestation/src/main/java/com/yubico/fido/metadata/FidoMetadataDownloader.java

```diff
@@ -148,6 +150,7 @@ public static class FidoMetadataDownloaderBuilder {
     private CertStore certStore = null;
     @NonNull private Clock clock = Clock.systemUTC();
     private KeyStore httpsTrustStore = null;
+    private boolean verifyDownloadsOnly = false;
```

Comment:
In a major version bump yes, certainly. But for a minor version this should be an opt-in feature, because this changes behaviour to a weaker security posture, strictly speaking.

Reply:
Or to put it another way: in this project I consider the [semantic versioning scheme](https://github.com/Yubico/java-webauthn-server/#semantic-versioning) to include changes in behaviour even if type signatures are unchanged.

Reply:
Sure thing, makes sense. Thank you!